tests/trustyai/utils.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+from ocp_resources.deployment import Deployment
+from ocp_resources.maria_db import MariaDB
+from ocp_resources.mariadb_operator import MariadbOperator
+from ocp_resources.pod import Pod
+from timeout_sampler import TimeoutSampler, TimeoutExpiredError
+from simple_logger.logger import get_logger
+
+TIMEOUT_5_MIN = 60 * 5
+
+LOGGER = get_logger(name=__name__)
+
+
+def wait_for_mariadb_operator_deployments(mariadb_operator: MariadbOperator) -> None:
+    expected_deployment_names: List[str] = [
+        "mariadb-operator",
+        "mariadb-operator-cert-controller",
+        "mariadb-operator-helm-controller-manager",
+        "mariadb-operator-webhook",
+    ]
+
+    for name in expected_deployment_names:
+        deployment = Deployment(name=name, namespace=mariadb_operator.namespace)
+        deployment.wait_for_replicas()
+
+
+def wait_for_mariadb_pods(mariadb: MariaDB, timeout: int = TIMEOUT_5_MIN) -> None:
+    namespace = mariadb.namespace
+    label_key = "app.kubernetes.io/instance"
+    label_value = "mariadb"
+
+    def _check_mariadb_pod() -> bool:
+        pods = Pod.get(namespace=namespace)
+        matching_pods = [pod for pod in pods if pod.labels.get(label_key) == label_value]
```

Comment:
```suggestion
matching_pods = [
        pod
        for pod in Pod.get(
            dyn_client=admin_client,
            namespace=namespace,
            label_selector="app.kubernetes.io/instance=MARIADB",
        )
    ]
```
do not forget to pass client